src/games/strategy/common/player/AbstractBaseAI.java

```diff
@@ -22,8 +23,8 @@ public AbstractBaseAI(final String name, final String type) {
   protected void pause() {
     try {
       ThreadUtil.sleep(AbstractUIContext.getAIPauseDuration());
-    } catch (final Exception ex) {
-      ex.printStackTrace();
+    } catch (final Exception e) {
```

Comment:
This `catch(Exception)` does not look necessary, ThreadUtil.sleep catches the Interrupted exception already.